packages/lending/src/protocols/morphoblue/lending-protocol.ts

```diff
@@ -135,25 +135,38 @@ export class LendingProtocol extends Protocol {
 
     const maxLtv = common.toBigUnit(lltv, 18);
 
+    const lstTokenAPYMap = await this.getLstTokenAPYMap(this.chainId);
+
+    const supplyApy = '0';
+    const supplyLstApy = lstTokenAPYMap[collateralToken.address.toLowerCase()] || '0';
+    const supplyGrossApy = supplyLstApy === '0' ? supplyApy : BigNumberJS(supplyApy).plus(supplyLstApy).toString();
```

Comment:
same as the `apy` comment

Reply:
added `calcSupplyGrossApy` and `calcBorrowGrossApy` to make the code clean